app/src/main/java/com/duckduckgo/app/notification/NotificationRegistrar.kt

```diff
@@ -65,7 +64,6 @@ class NotificationRegistrar @Inject constructor(
         const val PrivacyProtection = 101
         const val Article = 103 // 102 was used for the search notification hence using 103 moving forward
         const val AppFeature = 104
-        const val EmailWaitlist = 106 // 105 was used for the UOA notification
```

Comment:
@aitorvs QQ: Does this mean that going forward we can't use 106 as the notif id for backwards compatibility? Do we need to note that anywhere?

Reply:
Ah good catch. Will add that as a comment. 
We should probably make this more bullet proof than comments. Will prob create a separate task for that.